apps/site/assets/css/_footer.scss

```
@@ -58,38 +139,303 @@
 // MOBILE FOOTER STYLES
 //
 
-@include media-breakpoint-down(sm) {
+@include media-breakpoint-up(lg) {
   .m-footer__group {
-    display: none;
+    padding-left: 40px;
+  }
+}
+
+address {
+  .m-footer__item {
+    a {
+      color: $brand-primary;
+    }
+
+    @include media-breakpoint-down(xs) {
+      font-size: 14px;
+      line-height: 21px;
+    }
   }
+}
 
-  .m-footer__group-title {
-    display: none;
+.call-us-container {
+  margin-bottom: -50px;
+  background: $brand-primary-lightest-contrast;
+  padding-bottom: 0;
+  padding-top: 22px;
+  flex-basis: 30%;
+  border-radius: 50px 0 0 0;
+
+  .m-footer__links {
+    margin-bottom: 0px;
   }
 
-  .m-footer--mobile__group {
-    display: block;
+  .m-menu__feature {
+    background: $brand-primary-lightest-contrast;
+    font-size: 1rem;
+    margin-bottom: 14px;
+
+    &.emergency-contacts {
+      &.mobile {
+        border-radius: 0;
+        margin-top: 0px;
+        margin-bottom: 2px;
+  
+        @include media-breakpoint-up(lg) {
+          display: block;
+        }
+      }
+    }
+
+    @include media-breakpoint-down(sm) {
+      padding-top: 16px;
+      padding-bottom: 14px;
+      a {
+        margin-top: 20px;
+      }
+    }
+
+    @include media-breakpoint-down(xs) {
+      margin-bottom: 0px;
+      padding-top: 24px;
+    }
+
+    @include media-breakpoint-up(lg) {
+      padding-bottom: 26px;
+    }
+
+  small {
+    font-size: 16px;
+    @include media-breakpoint-down(xs) {
+      white-space: nowrap;
```

```suggestion
      font-size: 14px;
      white-space: nowrap;
```

Making font-size smaller on mobile to match the mocks.